### ISSUE TITLE
Current project in bolt is broken, can I import my working git repository to bolt/stackblitz to work with Bolt.new?

### ISSUE THREAD

**Issue by everspader** [stackblitz/bolt.new#367]
While working with Bolt.new I was creating the project in parallel in my local dev setup and committing and pushing to git.
After some prompt to restructure the project, destructive changes were applied in the Bolt environment and I couldn't rollback to the latest working version.

However, my local repository that was also pushed to git is in a good state and working version. Is it possible to import this repository back to Bolt and continue to work and give prompts on top of it?

**Comment by kirjavascript** [comment]
hey!

If the project is on Github you can open it in bolt by prepending `https://bolt.new/` to it, like `https://bolt.new/github.com/user/repo`.

free free to open another issue if you have any more questions

**Comment by everspader** [comment]
@kirjavascript thank you for the tip. It's worth mentioning that it only works if the repository is public even though my github account is already connected with Stackblitz to share all repos (public and private)

**Comment by kirjavascript** [comment]
We would like to support private repos for people in future! We're tracking this internally.

**Comment by kirjavascript** [comment]
that's a new problem! could you open a new issue for it?

**Comment by Labhk** [comment]
Hey, I have build this web tool for just this, previously using script on local now its a website much easy, enjoy
https://boltsync.mystify.tech/
https://www.youtube.com/watch?v=IneFM6ViV8s

**Comment by rapidstartup** [comment]
> Hey, I have build this web tool for just this, previously using script on local now its a website much easy, enjoy https://boltsync.mystify.tech/ https://www.youtube.com/watch?v=IneFM6ViV8s

1. WHAT.

This is awesome, bit worried about the security implications of giving you read/write access to all of our repos tho..you should opensource this (happy to pay for it) codebase for people to install their own so they have full ownership of the keys

**Comment by Labhk** [comment]
> > Hey, I have build this web tool for just this, previously using script on local now its a website much easy, enjoy https://boltsync.mystify.tech/ https://www.youtube.com/watch?v=IneFM6ViV8s
> 
> 1. WHAT.
> 
> This is awesome, bit worried about the security implications of giving you read/write access to all of our repos tho..you should opensource this (happy to pay for it) codebase for people to install their own so they have full ownership of the keys

Bro, It's safe I didn't add any backend in this all happening in client side, and github when you sign in gives does not give long term token, it gives you session token that expire after the session.
Any ways bolt folks might add this feature in few weeks. 
if you still need it, drop me message on discord @labhk or email labh.k2003@gmail.com

**Comment by everspader** [comment]
> Hey, I have build this web tool for just this, previously using script on local now its a website much easy, enjoy https://boltsync.mystify.tech/ https://www.youtube.com/watch?v=IneFM6ViV8s

This is amazing. It's a "basic" feature that should have been present in Bolt from the beginning.

**Comment by yashcrest** [comment]
@Labhk your link seems to not work anymore. Is there another working link ?

**Comment by Labhk** [comment]
Yes
boltsync.mystify.tech
Or
boltsync.vercel.app
